testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-kafka-ssl/kafka_schema_registry.properties

```diff
@@ -0,0 +1,13 @@
+connector.name=kafka
+kafka.nodes=kafka:9092
+kafka.table-description-supplier=confluent
+kafka.confluent-schema-registry-url=http://schema-registry:8081
+kafka.default-schema=product_tests
+
+kafka.security-protocol=SSL
+kafka.keystore-file      =/docker/presto-product-tests/conf/presto/etc/catalog/secrets/kafka.client.keystore.jks
+kafka.keystore-password  =confluent
+kafka.truststore-file    =/docker/presto-product-tests/conf/presto/etc/catalog/secrets/kafka.client.truststore.jks
+kafka.truststore-password=confluent
+kafka.key-password       =confluent
+kafka.endpoint-identification-algorithm=disabled
```

Comment:
please fix indentation (remove spaces before =)

Reply:
Sorry I didn't rebase my commits back then. I wasn't aware of co-author GH feature. I just merged both commits.